package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skpm/util",
-  "version": "0.1.18",
+  "version": "0.1.19",
```

Comment:
you don't need to bump it, I already published it and updated Sketch to pull it 👍